apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/SimpleManagedTracer.cs

```diff
@@ -203,6 +206,14 @@ private void EndSpan(Span span)
 
             lock (_traceLock)
             {
+                // If the time between the start and end of a span is less then 1 ms
+                // the span will not be recorded.  If this is the case bump up
+                // the time by 1 ms which is the smallest amount of time for the Trace API
+                // to record a trace. 
+                if ((span.TraceSpan.EndTime.Nanos - span.TraceSpan.StartTime.Nanos) < _nanosecondsInAMillisecond)
```

Comment:
Is span.TraceSpan.EndTime a Timestamp? If so, this won't be right - the nanos value could be out of range, and there could be a whole day of difference anyway. (Additionally, it won't catch the case where the two timestamps are less than a millisecond apart, but on different days.)

If it *is* a Timestamp, it might be worth assigning a bug to me and I'll look at it tomorrow.